extensions/languagetool/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "include": ["src/**/*", "raycast-env.d.ts"],
+  "compilerOptions": {
+    "lib": ["ES2023"],
+    "module": "commonjs",
```

Comment:
You could change it to ESM since you specified the module type to `module` in the `package.json`.

Here is an example: https://github.com/raycast/extensions/blob/fa209f5949bf563fcb0a3fc7e686744c1a0023e4/extensions/say/tsconfig.json